plugins/convention/src/main/java/org/apache/struts2/convention/PackageBasedActionConfigBuilder.java

```diff
@@ -673,17 +697,17 @@ protected void buildConfiguration(Set<Class> classes) {
             for (String namespace : namespaces) {
                 String defaultActionName = determineActionName(actionClass);
                 PackageConfig.Builder defaultPackageConfig = getPackageConfig(packageConfigs, namespace,
-                        actionPackage, actionClass, null);
+                    actionPackage, actionClass, null);
 
                 // Verify that the annotations have no errors and also determine if the default action
                 // configuration should still be built or not.
                 Map<String, List<Action>> map = getActionAnnotations(actionClass);
                 Set<String> actionNames = new HashSet<>();
                 boolean hasDefaultMethod = ReflectionTools.containsMethod(actionClass, DEFAULT_METHOD);
                 if (!map.containsKey(DEFAULT_METHOD)
-                        && hasDefaultMethod
-                        && actionAnnotation == null && actionsAnnotation == null
-                        && (alwaysMapExecute || map.isEmpty())) {
+                    && hasDefaultMethod
+                    && actionAnnotation == null && actionsAnnotation == null
+                    && (alwaysMapExecute || map.isEmpty())) {
```

Comment:
My personal preference is to use double indents for continuations - it avoids situations like these where the guard blends in with the following code

Reply:
I'm using standard IDEA settings, yet it would be good to introduce a common IDE independent mechanism to format code.

Reply:
IMHO Eclipse does this by default, IntelliJ not.

We could introduce `.editorconfig` settings. The following setting does the job:

```
[*.java]
continuation_indent_size = 8
```

Reply:
Just checked this as I was curious and doesn't seem to be the case (at least on the version I'm using)
![image](https://github.com/apache/struts/assets/38072764/bc03228d-4f2e-4e81-9999-42af58a521e3)

Reply:
Oh, right! Just checked my settings and it was 4 for continuation, reverted to default one :)